docs/zkapps/o1js/permissions.mdx

```diff
@@ -331,27 +416,6 @@ await tx.prove();
 await tx.sign([feePayerKey, zkappKey]).send();
 ```
 
-## Upgrading after an update to the Mina Protocol
```

Comment:
moved this above and add some changes to give more context before we dive into examples